README.md

```diff
@@ -6,6 +6,14 @@ You can find the collection of all [Bitrise integrations](https://www.bitrise.io
 
 If you find something missing from the steps, you can drop us an issue, or [create your own step](http://devcenter.bitrise.io/bitrise-cli/create-your-own-step). See our example for creating & sharing a new step under [/step-template](https://github.com/bitrise-steplib/step-template).
 
+### Important
+
+One PR should only contain one file (except the icon), and one change.
```

Comment:
I think it should be - `should only contain -> should contain only` - here and in the 15th line too.

Reply:
Fixed.